ape_trezor/client.py

```diff
@@ -102,80 +108,58 @@ def sign_personal_message(self, message: bytes) -> Tuple[int, bytes, bytes]:
         ethereum_message_signature = sign_message(
             self.client, self._account_hd_path.address_n, message
         )
-
         return extract_signature_vrs_bytes(signature_bytes=ethereum_message_signature.signature)
 
-    # TODO: Uncomment when Trezor has released the EIP 712 update
-    # def sign_typed_data(self, domain_hash: bytes, message_hash: bytes)
-    # -> Tuple[int, bytes, bytes]:
+    # def sign_typed_data(
```

Comment:
I have another branch that uncomments this; however my device isn't liking it yet, at least not with the default HD Path.